core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -101,7 +102,7 @@ object ConsoleConsumer extends Logging {
       } catch {
         case e: Throwable => {
           error("Error processing message, stopping consumer: ", e)
```

Comment:
Can you change this log message slightly? This isnt closing the consumer directly